app/models/friendship.rb

```diff
@@ -0,0 +1,67 @@
+class Friendship < ActiveRecord::Base
+  extend TokenContext
+  include Tokenizer
+	enum friendship_status: [ :denied, :pending, :accepted]
+  attr_accessor :friendship_request_token
+	cattr_accessor :daily_request_limit
```

Comment:
I do not see any typo here. My indentation is also fine in my editor, I don't know why it is looking different here.
The cattr_accessor is meant to do what it does - provides access to a class variable that prevents users from adding more than 12 users. It is used in user.rb line 68 and friendship.rb line 21